### ISSUE TITLE
Monailabel launches 2 process on linux 

### ISSUE THREAD

**Issue by AHarouni** [Project-MONAI/MONAILabel#1167]
When running monailabel command then do a 
```
ps  -aex |grep label 
 594 ?        S      0:00 /bin/bash /opt/conda/bin/monailabel start_server -p 44499 --app apps/radiology --studies datasets/Task09_Spleen/imagesTr --conf models deepedit,segmentation,deepgrow
  607 ?        Sl     0:18 python -m monailabel.main start_server -p 44499 --app apps/radiology --studies datasets/Task09_Spleen/imagesTr --conf models deepedit,segmentation,deepgrow_3d,deepgr
  693 pts/0    S+     0:00 grep --color=auto label NPP_VERSION=11.7.4.75 SHELL=/bin/bash NVIDIA_VISIBLE_DEVICES=all DALI_BUILD=5322998 CUSOLVER_VERSION=11.4.0.1 NVM_INC=/usr/local/nvm/versions
```
As you see this opens 2 process which can be confusing when trying to kill it since we need to run it in background 

Simple fix is to add `exec` to the last line of 
https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/scripts/monailabel#L28
as 
```
exec ${PYEXE} -m monailabel.main $*
```

**Comment by SachidanandAlle** [comment]
One is bash command and other is python.. bash is the the parent process... You can kill either.. but if you kill parent kill full tree (kill -9)

**Comment by SachidanandAlle** [comment]
Good suggestion.  It will be fixed as part of #1208
